.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
+labels:
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- HW Platform (e.g. iPhone 8, Pixel 2, Samsung Galaxy, Intel based Mac, i7 based Dell with NVIDIA GPGPU etc., or Google Cloud machine)
+- TensorFlow installed from (source or binary):
+- TensorFlow version (use command below):
+- TensorFlow Addons version:
+- Python version and type(eg. Anaconda Python, Stock Python as in Mac, or homebrew installed Python etc):
+- Bazel version (if compiling from source):
+- GCC/Compiler version (if compiling from source):
+- CUDA/cuDNN version (if compiling/running with GPGPU):
```

Comment:
This has always been a troublesome ask from users. How do we feel about asking for this information if the issue is found to be related to a cuda kernel or something? It's a bit of burden on the addons team, but not really that much

Reply:
Hmmm, why was this marked resolved? I still think this is a tough question for many to answer.

Reply:
@seanpmorgan but maybe we can think of some simpler ones or just make the user run some script from which we will get all this info

Reply:
Good idea. For now I think we can leave this off and see what TF core ends up with. Worst case scenario we can always ask someone to get the info after they start the issue.

Reply:
Cool then let's resolve this conv

Reply:
Haha, sorry miscommunication. We can resolve it by removing this bullet. I'd like to try to field bug reports without this until we have a stable and easy way to get this information from the user.